### ISSUE TITLE
Allow reordering of experiment metrics

### ISSUE THREAD

**Issue by aaronyan** [Automattic/abacus#834]
What do y'all think about allowing experimenters to reorder the metrics in an experiment results page? This request was originally created here: pbmo2S-1Tg-p2#comment-2811

We don't have a place to easily store this data, so quite a few changes would need to be made. I think there would be lots of other features that get unlocked by adding a database specific to Abacus user preferences:

- Favorited experiments and metrics
- Experiment planning (dates, queuing)
- User groups / security etc.

### To do

- [ ] Discuss if reordering metrics is worth adding
- [ ] Come up with a plan to add reordering
- [ ] Implement reordering of metrics

**Comment by jessie-ross** [comment]
> We don't have a place to easily store this data, so quite a few changes would need to be made. I think there would be lots of other features that get unlocked by adding a database specific to Abacus user preferences:

Yep, this is a big one blocking a lot of features, we need a KV store with an API.
